insights/parsers/awx_manage.py

```diff
@@ -9,14 +9,16 @@
 """
 
 from insights import JSONParser, parser, CommandParser
-from insights.parsers import SkipException, ParseException
+from insights.parsers import SkipException
 from insights.specs import Specs
+import json
 
 
 @parser(Specs.awx_manage_check_license)
 class AnsibleTowerLicenseType(CommandParser, JSONParser):
     """
     Parses the output of command  ``awx-manage check_license``
+    NOTE: This class is deprecated, consider to use class AnsibleTowerLicense
```

Comment:
Better use the following format
```
.. note::

    This ...
```

See https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html